src/Tizen.NUI/src/internal/EXaml/Action/CallAddMethodAction.cs

```diff
@@ -64,10 +64,10 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && childOp.ValueList.Count > 2 && (childOp.ValueList[0] is Instance instance) && (childOp.ValueList[1] is Instance instance1))
             {
-                int parentIndex = (childOp.ValueList[0] as Instance).Index;
-                int childIndex = (childOp.ValueList[1] as Instance).Index;
+                int parentIndex = instance.Index;
+                int childIndex = instance1.Index;
```

Comment:
`instance1` naming doesn't look good.
isn't it like this?
```suggestion
                int childIndex = child.Index;
```

Reply:
done